### PR TITLE
Revamp guns: Revival

### PR DESCRIPTION
**Pull request by TaralGit**
Again, replacing the cev eris sprites and purging some unused sprites. Should be ready in 2 days max.

revamp
other_revamp

### CODE REVIEW

**Comment by TaralGit** [comment]
Ok, this is ready for review.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Rifles/rifles.yml [hunk 1]
```diff
@@ -87,9 +87,9 @@
   - type: Appearance
 
 - type: entity
-  name: Z8 Bulldog
+  name: M-90gl
   parent: BaseWeaponRifle
-  id: WeaponRifleBulldog
+  id: WeaponRifleM90
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Rifles/rifles.yml:92]: Should probably be "M-90 grenade launcher" and WeaponRifleM90GrenadeLauncher

File: Resources/Textures/Objects/Weapons/Guns/Battery/laser_gun.rsi/meta.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from cev eris at https://github.com/discordia-space/CEV-Eris/blob/master/icons/obj/guns/energy/retro.dmi",
+    "copyright": "Taken from ds13 at https://github.com/DS-13-Dev-Team/DS13/blob/dev/icons/obj/gun.dmi",
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Weapons/Guns/Battery/laser_gun.rsi/meta.json:4]: Should have a specific commit (yeah the old one also had this issue).

File: Resources/Textures/Objects/Weapons/Guns/Battery/makeshift.rsi/meta.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from cev eris at https://github.com/discordia-space/CEV-Eris/blob/master/icons/obj/guns/energy/makeshift_carbine.dmi",
+    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/blob/master/icons/obj/guns/energy.dmi",
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Weapons/Guns/Battery/makeshift.rsi/meta.json:4]: Same here

File: Resources/Textures/Objects/Weapons/Guns/Shotguns/bulldog.rsi/meta.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken/modified from cev-eris at https://github.com/discordia-space/CEV-Eris/blob/master/icons/obj/guns/projectile/lamia.dmi",
+    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/blob/master/icons/obj/guns/ballistic.dmi",
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Weapons/Guns/Shotguns/bulldog.rsi/meta.json:4]: commit here

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
FYI icon sprites aren't generally needed anymore, spriteview uses dummy appearance data.

**Comment by ike709** [comment]
> There's a problem: the reason why I didn't linked specific commits is because I couldn't find them (also, yay, more conflicts). As in I couldn't find any of the original old commits that first introduced the bulldog shotgun or the laser rifle from ds13.

You don't need the original commit that added a sprite, in fact you *shouldn't* use that one since sprites can change. You just need the commit you took the sprite from.

File: Resources/Textures/Objects/Weapons/Guns/Shotguns/bulldog.rsi/meta.json (hunk 4 above)
Comment (TaralGit) [reply Resources/Textures/Objects/Weapons/Guns/Shotguns/bulldog.rsi/meta.json]: Done

File: Resources/Textures/Objects/Weapons/Guns/Battery/makeshift.rsi/meta.json (hunk 3 above)
Comment (TaralGit) [reply Resources/Textures/Objects/Weapons/Guns/Battery/makeshift.rsi/meta.json]: found the initial commit for the file

File: Resources/Textures/Objects/Weapons/Guns/Battery/laser_gun.rsi/meta.json (hunk 2 above)
Comment (TaralGit) [reply Resources/Textures/Objects/Weapons/Guns/Battery/laser_gun.rsi/meta.json]: found the initial commit on the Polaris server

**Comment by TaralGit** [comment]
ok, everything is good here. no conflicts, all checks passed.

**Pull request merged**